### ISSUE TITLE
conda install missing activate/deactivate scripts

### ISSUE THREAD

**Issue by rajatgirotra** [conda/conda#4381]
I have just installed conda and I see there are no activate/deactivate scripts in miniconda2/bin folder
This is causing all my recipes to fail as they try to do 

> source /home/ubuntu/miniconda2/bin/activate

```
ubuntu@ubuntu-xenial:~/miniconda2/bin$ ls /home/ubuntu/miniconda2/bin/activate
ls: cannot access '/home/ubuntu/miniconda2/bin/activate': No such file or directory
```

Here's my conda info:

ubuntu@ubuntu-xenial:/vagrant/recipes_ubuntu/python_base$ conda info

```
Current conda install:

               platform : linux-64
          conda version : 4.3.5
       conda is private : False
      conda-env version : 2.6.0
    conda-build version : 2.1.1
         python version : 2.7.13.final.0
       requests version : 2.12.4
       root environment : /home/ubuntu/miniconda2  (writable)
    default environment : /home/ubuntu/miniconda2
       envs directories : /home/ubuntu/miniconda2/envs
          package cache : /home/ubuntu/miniconda2/pkgs
           channel URLs : file:///home/ubuntu/conda_builds/linux-64
                          file:///home/ubuntu/conda_builds/noarch
                          https://repo.continuum.io/pkgs/free/linux-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/r/linux-64
                          https://repo.continuum.io/pkgs/r/noarch
                          https://repo.continuum.io/pkgs/pro/linux-64
                          https://repo.continuum.io/pkgs/pro/noarch
            config file : /home/ubuntu/.condarc
           offline mode : False
             user-agent : conda/4.3.5 requests/2.12.4 CPython/2.7.13 Linux/4.4.0-59-generic debian/stretch/sid glibc/2.23
                UID:GID : 1000:1000
```

conda-env update is also isn't helping

```
ubuntu@ubuntu-xenial:~/miniconda2/bin$ conda update conda-env
Fetching package metadata ...........
Solving package specifications: .

# All requested packages already installed.
# packages in environment at /home/ubuntu/miniconda2:
#
conda-env                 2.6.0                         0    defaults

```

**Comment by kalefranz** [comment]
We've just released 4.3.6, which fixes that problem.  Sorry for the trouble.

**Comment by kalefranz** [comment]
We released 4.3.6 this afternoon that fixes this problem. Just `conda update conda`.
